packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/review/arm-networkanalytics.api.md

```diff
@@ -63,7 +63,7 @@ export type CreatedByType = string;
 
 // @public
 export interface DataProduct extends TrackedResource {
-    identity?: ManagedServiceIdentity;
+    identity?: ManagedServiceIdentityV4;
```

Comment:
this is updated the typespec with rest api spec latest main. But I think we should confirm what the real current behavior is and fix the spec instead.